hyperspy/roi.py

```diff
@@ -580,17 +610,20 @@ class Point1DROI(BasePointROI):
     value = t.CFloat(t.Undefined)
     _ndim = 1
 
-    def __init__(self, value):
+    def __init__(self, value=None):
         super(Point1DROI, self).__init__()
+        value = value if value is not None else t.Undefined
         self.value = value
 
-    def __getitem__(self, *args, **kwargs):
-        _tuple = (self.value,)
-        return _tuple.__getitem__(*args, **kwargs)
-
+    def _set_default_values(self, signal):
+        ax0, *_ = self._parse_axes(None, signal.axes_manager)
+        # If roi parameters are undefined, use center of axes
+        if t.Undefined in list(self):
```

Comment:
Are you suggesting that we "overwrite" the ROI parameters if they are set? I think there is value in being able to give a ROI non-default parameters (for instance, from a different previous ROI) and be able to "take a look" interactively to see where it sits - especially since we have no "non-interactive" ROI plot functionality!

Reply:
I am suggesting that the method does what its name implies.
> I think there is value in being able to give a ROI non-default parameters (for instance, from a different previous ROI) and be able to "take a look" interactively to see where it sits - especially since we have no "non-interactive" ROI plot functionality!

I don't understand how does it relate with the `_set_default_values` method here?

Reply:
I totally misunderstood your intent. Agreed and fixed.